linux/map.jinja

```diff
@@ -126,6 +132,12 @@
         'bridge': 'none',
         'proxy': {
            'host': 'none',
+           'apt': {
+              'enabled': 'True'
```

Comment:
Its enabled, to keep backward compatibility with previous implementations, where only proxy:host and proxy:port was defined. These default True is a must.